### PR TITLE
feat(v2): v2 of plugins

### PR DESCRIPTION
**Pull request by Flaxoos**
V2 introduces a crucial change to the build, as it was using a fat jar so was problematic to use

Added app.yaml/conf

### CODE REVIEW

**Comment by bjhham** [comment]
Hey, there are some conflicts from some maintenance I was doing earlier.  I'll revert them now so these changes can override.

File: plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/application.conf [hunk 1]
```diff
@@ -0,0 +1,30 @@
+ktor {
```
Comment (bjhham) [inline plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/application.conf:1]: Looks like you'll need some lines in the manifest file:

```
installation:
  default: install.kt
  application_conf: application.conf
  application_yaml: application.yaml
```

**Review by bjhham — CHANGES_REQUESTED** [review]
Looks good besides the manifest lines for the config files.

Please try out a test project with `./gradlew buildTestProject --args="--plugins=ktor-server-kafka-jvm,ktor-server-task-scheduling"` to see if it generates correctly, and let me know if you run into any problems.

File: plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/application.conf (hunk 1 above)
Comment (Flaxoos) [reply plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/application.conf:1]: Addressed and added another plugin

File: plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/documentation.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# Ktor Server Kafka
+
+![Build Status](https://img.shields.io/github/actions/workflow/status/flaxoos/extra-ktor-plugins/build-and-publish-main.yml?event=push&logo=githubactions&style=for-the-badge)
+[![Maven-central](https://img.shields.io/maven-central/v/io.github.flaxoos/ktor-server-task-scheduling?style=for-the-badge&logo=apachemaven)](https://search.maven.org/search?q=io.github.flaxoos%20AND%20ktor)
+![Language: Kotlin](https://img.shields.io/github/languages/top/flaxoos/flax-ktor-plugins?color=blue&logo=kotlin&style=for-the-badge)
+[![Koverage: 94.42%](https://img.shields.io/badge/94.42-green?logo=kotlin&label=koverage&style=for-the-badge)](file:/Users/ido/IdeaProjects/flax-ktor-plugins/build/reports/kover/html/index.html)
+[![Docs](https://custom-icon-badges.demolab.com/badge/Pages-blue.svg?label=Docs&logo=github&logoColor=white?icon=githubpages&style=for-the-badge)](https://github.com/Flaxoos/extra-ktor-plugins/actions/workflows/pages/pages-build-deployment)
+[![Awesome Kotlin Badge](https://custom-icon-badges.demolab.com/badge/awesome-kotlin-orange.svg?labelColor=blue&style=for-the-badge)](https://github.com/KotlinBy/awesome-kotlin)
```
Comment (bjhham) [inline plugins/server/io.github.flaxoos/ktor-server-kafka-jvm/2.0/documentation.md:8]: Can you drop the title and badges here?  The title will be redundant in the generator and badges won't render.

**Pull request merged**